index.html

```diff
@@ -10,163 +10,287 @@
   </head>
   <body>
     <header>
-      <h1 class="heading">Quiz-App</h1>
-      <h2 class="heading hidden-js">Bookmarks</h2>
-      <h2 class="heading hidden-js">Create</h2>
-      <h2 class="heading hidden-js">Profile</h2>
+      <h1 data-js="home-title" class="heading">Quiz-App</h1>
```

Comment:
If you toggle this to hidden-js, there is no h1 on the page anymore. You should make the other headings h2 as well.